datadog_checks_base/datadog_checks/utils/__init__.py

```diff
@@ -1,3 +1,6 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+from ddtrace import patch
+
+patch(requests=True)
```

Comment:
when the Python interpreter is invoked from the Agent, do we know what integrations will be loaded? We may try to find a better place to patch enabled checks.

Reply:
@masci Is this something you are familiar with? I can look into digging up the answer if not. 

Do we know what the overhead of enabling this early on for all checks would be?